init.lua

```diff
@@ -75,14 +90,36 @@ function diet.item_eat(max)
 				return
 			end
 		end
-		
+
 		-- Increase health
-		if minetest.get_modpath("hud") and hud then
-			local h = tonumber(hud.hunger[name])
+		if minetest.get_modpath("hbhunger") and hbhunger then
```

Comment:
Oh yeah ;) it's done, but i haven't really tested with it (the old hud) because i don't find it anywhere. But logically this may work.